### ISSUE TITLE
Reproducing Results on CelebAMask-HQ

### ISSUE THREAD

**Issue by yuval-alaluf** [NVIDIA/pix2pixHD#209]
Hi,  
I am running label-to-image using the CelebAMask-HQ dataset similar to the work in your paper on the Helen Faces dataset.   
However, it appears that the model is able to almost completely reproduce the original image (including the background of the original image) even though the input to the generator is the segmentation map.  
Further, when performing inference on the test set it appears the generated image is nearly identical to the real image.  
While it makes sense that the resulting image may be similar in **structure** to the original, the nearly identical reconstruction of the background hints that I am missing something here.
For example, below is a training sample's segmentation map, real image, and output image:
![Screen Shot 2020-07-20 at 10 18 56 AM](https://user-images.githubusercontent.com/37438467/87910378-9437f880-ca72-11ea-8cbe-9bda649afb48.png)
And a result on a test image (I don't have the corresponding real image):
![Screen Shot 2020-07-20 at 10 19 42 AM](https://user-images.githubusercontent.com/37438467/87910389-9d28ca00-ca72-11ea-82fe-9c43f7737284.png)

Here are the arguments used: 
--gpu_ids 0   
--name celebs_256_seg2face
--checkpoints_dir /..../experiment/path  
--batchSize 24   
--loadSize 256   
--fineSize 256   
--label_nc 19   
--input_nc 3  
--output_nc 3   
--dataroot /.../CelebAMask-HQ/  
--no_instance   
--niter 50   
--niter_decay 50   
--nThreads 8

I verified that the input label is indeed the segmentation map which is transformed into a one-hot vector of size 19. I also verified that none of the test images appear in the training set.  

I appreciate any help in understanding what I may be missing.  
Thanks in advance!

**Comment by prajwaljpj** [comment]
Were you able to reproduce the results?
